### PR TITLE
TRUNK-5409: Implement DosingInstructions to ChemoDosingInstructions

### PR DESCRIPTION
**Pull request by IamMujuziMoses**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5409

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by IamMujuziMoses** [comment]
@wluyima @dkayiwa @ibacher could you please review this PR

**Review by mherman22 — COMMENTED** [review]
Thanks @IamMujuziMoses for the commit. Find the comments below

File: api/src/main/java/org/openmrs/ChemoDosingInstructions.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import java.util.Date;
+import java.util.Locale;
+import java.util.StringTokenizer;
+
+import org.openmrs.api.APIException;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+
+/**
+ * @since 1.10
```
[the remark this answers is not in the document]
Comment (IamMujuziMoses) [reply api/src/main/java/org/openmrs/ChemoDosingInstructions.java]: @mherman22 thanks for that, fixed it

**Review by njiddasalifu — APPROVED** [review]
This looks good to me. Though i am still to gain much experience in Java

**Pull request closed without merging**